test/ruby/signature/test_test.rb

```diff
@@ -3,6 +3,8 @@
 require "ruby/signature/test"
 require "logger"
 
+return unless Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('2.7.0')
```

Comment:
It skips test for `Ruby::Signature::Test:Hook` if the Ruby runtime version is less than 2.7.

We also use ruby2_keywords gem to apply the polyfill instead of the skipping, but it needs https://github.com/ruby/ruby2_keywords/pull/3.